src/macaron/malware_analyzer/pypi_heuristics/metadata/wheel_absence.py

```diff
@@ -72,7 +75,24 @@ def analyze(self, pypi_package_json: PyPIPackageJsonAsset) -> tuple[HeuristicRes
                 if release_metadata["packagetype"] == self.WHEEL:
                     wheel_present = True
 
-                release_files.append(release_metadata["filename"])
+                name = json_extract(pypi_package_json.package_json, ["info", "name"], str)
+                if name is None:
+                    error_msg = "There is no 'name' field for this package."
+                    logger.debug(error_msg)
+                    raise HeuristicAnalyzerValueError(error_msg)
+
+                # include the pypi inspector link, which uses the same suffix of
+                # packages/{blake2b_256}/file_name
+                inspector_prefix = f"{self.INSPECTOR_PREFIX}{name.lower()}/{version}/"
+                inspector_link = release_metadata["url"].replace(self.PYPI_PREFIX, inspector_prefix)
+
+                # use a head request because we don't care about the response contents
+                if send_head_http_raw(inspector_link) is None:
+                    inspector_link = ""
+
+                release_files.append(release_metadata["url"])
```

Comment:
Can you please add separate fields in a `dict` for the inspector and file_server instead of adding all to a `list`?

Also, we could add a separate field for version instead of using the version as the key for the return value of this heuristic.

Reply:
I've done three fields, `version`, `file_server_links`, and `inspector_links`. I agree that makes much more sense, and allows me to resolve an invalid pypi inspector URL much easier.